_dev/docker/ci/Dockerfile

```
@@ -33,7 +32,9 @@ RUN yarn install --immutable;
 # must based on cimg/node:20.11-browsers, which is why this stage
 # is necessary.
 FROM builder as playwright-install
-RUN npx playwright install firefox;
+RUN npx playwright install --with-deps firefox \
+ && npx playwright install --with-deps chromium \
+ && npx playwright install --with-deps webkit;
```

This seems verbose and i wonder if a playwright installation is attempted multiple times. Would something like the following work? 🤔 
```suggestion
RUN npx playwright install --with-deps firefox chromium webkit;
```

The [documentation](https://playwright.dev/docs/browsers#install-system-dependencies) also makes me wonder if '--with-deps' can be used standalone?

---

Yes you are right. I will change it.